packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -5,7 +5,12 @@
     :class="['card-container', addClass]"
   >
     <span class="morph">
-      <button :class="['morph-display-wrapper', 'btn', btnType]" @click="open()">
+      <button
+        :class="[isSeamless ? 'morph-display-wrapper-seamless': 'morph-display-wrapper',
+                 'btn',
```

Comment:
Shouldn't this button has the `morph-display-wrapper` anyway regardless of whether it is seamless or not?

Reply:
I added the new `morph-display-wrapper-seamless` since I've noticed that the colour of the minimized panels would not inherit the parent's colour with just `morph-display-wrapper`

With just `morph-display-wrapper`:

https://github.com/MarkBind/markbind/assets/105090139/3bf36d36-47ae-40d8-a6fa-5054a9eef68c

With `morph-display-wrapper-seamless` and `morph-display-wrapper`

https://github.com/MarkBind/markbind/assets/105090139/ef8dbacb-5b01-49ff-ad9c-e8dc54e25af7

Reply:
As in does the a seamless button also need the properties from `morph-display-wrapper`?

Reply:
No seamless buttons wouldn't need properties from `morph-display-wrapper`, seamless buttons only appear when the panel is not minimised so it shouldn't need any properties from `morph-display-wrapper`

Reply:
I've taken a video for your reference that shows that toggling these properties would make a visual difference. The caret button seems to also be a seamless button which appears when the panel is minimised, so the non seamless panels and the seamless panels then have slightly different styling due to this line of code. 

https://github.com/MarkBind/markbind/assets/59471320/8be20b60-03e2-4759-89e2-481bb39f2094

Reply:
Rather than doing this or else statement, could all buttons have `morph-display-wrapper` by default and then only seamless buttons will have `morph-display-wrapper-seamless`?

Reply:
yep sure I've implemented it as such now